app/services/api/concerns/filter_ignorable.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+module API
+  module Concerns
+    module FilterIgnorable
+      extend ActiveSupport::Concern
+
+      def ignore?(filter:)
+        filter == :ignore || (!filter.nil? && filter.blank?)
+      end
+    end
+  end
+end
```

Comment:
I'm just not sure about the location of this concern, no reason other than "looks weird" lol

Maybe `app/services/concerns` is better location, that way we can use `include FilterIgnorable`